Dockerfile

```diff
@@ -1,17 +1,22 @@
-FROM python:3.7
+FROM fedora:38
 
 ENV APP_PORT=${APP_PORT:-5700}
 
-RUN python3 -m pip install --upgrade pip && \
-    python3 -m pip install --upgrade setuptools wheel
+RUN dnf --assumeyes makecache && \
+    dnf --assumeyes install \
+        python3 \
+        python3-pip \
+        python3-pysvn \
+        ftp
```

Comment:
1. If we are encrypting with binary *gpg* we should install it here too.
2. Reason for installing *ftp* seems not obvious. If we are using FtpFS then it is not necessary.

Reply:
GPG comes with fedora:38, but ok, no problem to add.